components/Tabs/src/MDCTabBar.h

```diff
@@ -93,6 +94,14 @@ IB_DESIGNABLE
 /** Ink color for taps on tab bar items. Default: Semi-transparent white. */
 @property(nonatomic, nonnull) UIColor *inkColor UI_APPEARANCE_SELECTOR;
 
+/**
+ Ink style for taps on tab bar items.
+
+ Defaults to MDCInkStyleBounded when UIBarPosition is Top or Any.
+ Defaults to MDCInkStyleUnbounded when UIBarPosition is Bottom.
+*/
+@property(nonatomic, assign) MDCInkStyle inkStyle;
```

Comment:
I think we should just hard-code the behavior internally and not allow overwriting the property via a public API.

Reply:
I spoke with our designers and confirmed that internal behavior should align to the Material.io guidelines for motion, but the colors would be appropriately changed for theming.

Here's the appearance of Tabs Ripple in Material: https://material.io/design/components/tabs.html#behavior